test/Makefile

```diff
@@ -1,24 +1,29 @@
-REPO := $(shell basename $(shell git remote get-url origin) .git)
-CONTAINER := $(REPO)_test
+ACCOUNT := 224588347132
+CONTAINER := terraform_modules_test
```

Comment:
I think we should make sure the CONTAINER name is unique. I suggest it be `$(REPO)-tests`.

Reply:
This is a great suggestion.

During earlier test runs, I forgot to rebuild the image when moving from module to module, so this would stop that from happening.